### ISSUE TITLE
Tutorial vs Examples

### ISSUE THREAD

**Issue by shantenujha** [radical-cybertools/radical.pilot#861]
I thought we agreed to: 

 - user guide would be simple possibly not stand-alone examples, so user get a guide of important and basic functionality via examples

 - section 5 would be "examples" which would be local and not assume/require remote access to machines.

 - wouldn't "examples" like k-means etc be therefore in section 5 and not in 4?

I remember we having agreed to these too!!

**Comment by andre-merzky** [comment]
This is already done, but not pushed into a release, together with quite a number of other changes.  We'll do one more release tonight, which then also contains the reservation and allocations we want to use, in the respective configs.  We are still waiting on the slurm reservation ID.

For a somewhat more up-to-date version (not fully though), please see http://radicalpilot.readthedocs.org/en/release-v0.37/

**Comment by vivek-bala** [comment]
> which then also contains the reservation and allocations we want to use, in the respective configs

Shouldn't all the user guide scripts be independent (/stand-alone) of the machine. I thought they were to be run locally (in which case why does one need allocation-ids). Am I missing something ? (trying to keep things consistent between enmd and rp)

**Comment by andre-merzky** [comment]
Yes, things are supposed to run locally for the user guide and tutorial part.  We use some of our config  files also for the excercises though, and want to put those configs into the release.  That may be RP specific though...

**Comment by vivek-bala** [comment]
okk.. thanks..

**Comment by andre-merzky** [comment]
SJ: to clarify, the UG is still using stand alone examples -- but each of them focuses on *exactly one* small and well defined capability.  Apart  from the basic example, which is kind of the minimal RP app one can sensibly write, which is introduced piecewise in 'getting started'.
